### DISCUSSION TITLE
Relation managers inside relation managers

### DISCUSSION THREAD

**Discussion by Saad5400** [filamentphp/filament#14904] [Ideas]
Adding nested resources and/or what I think would be very useful: the ability to use relation managers inside relation manager's record edit/view modals.

Currently I'm doing something like this: [nested-resources](https://laraveldaily.com/post/filament-v3-nested-resources-trait-pages), but it would be so much better if it is supported by filament directly.

**Comment by leandrocfe** [comment]
https://github.com/filamentphp/filament/issues/7104

**Reply by Saad5400** [reply]
Thanks, but I'm aware of this. I was suggesting to support it officially within Filament.

Because doing as the linked discussion has quite some boilerplate and is just not very clean.

However rendering relation managers inside of view/edit modals of a relation manager record, would be very clean and easy to implement (I think).
